Gallery page ready, new page for Hacktoberfest form & some fix 🔃

daoodaba975 opened the pull request.

daoodaba975 committed:
🧩 :: load new Translations.

daoodaba975 committed:
🔃 :: update Translations.

daoodaba975 committed:
✅ :: add Photos source & update.

daoodaba975 committed:
🎉 :: new page for Hacktoberfest.

daoodaba975 committed:
✅ :: fix error.

NDOY3M4N commented on src/components/Header.tsx:
```diff
@@ -44,13 +45,13 @@ function Header(props: HeaderProps) {
+				src="https://beamanalytics.b-cdn.net/beam.min.js"
+				data-token={process.env.BEAM_DATA_TOKEN}
+				async
+			/>
```
Is this working now? We'll have to see the deployment I guess.

daoodaba975 replied:
I want to test it again that's why I put it back outside of the Head

NDOY3M4N commented on src/components/gallery/EventsGallery.tsx:
```diff
@@ -23,137 +25,165 @@ const EventsGallery = () => {
 			</div>
 			<div className="grid grid-cols-1 gap-4 lg:grid-cols-2 lg:gap-16">
```
maybe we could `md:grid-cols-2` instead of `lg:grid-cols-2` since we have enough empty space in medium breakpoint. We could also decrease the `gap`.

NDOY3M4N replied:
I added some modification to this section. You can check it if you have some time.

daoodaba975 replied:
Nice 👍🏾

NDOY3M4N commented on src/pages/hacktoberfest23.tsx:
```diff
@@ -0,0 +1,18 @@
+import Header from '@/components/Header';
+import Script from 'next/script';
+
+export default function About() {
```
for this page maybe we should add a custom container because the form banner is cutoff on smaller devices.

NDOY3M4N replied:
![image](https://github.com/GalsenDev221/website/assets/46305144/65dd8c97-c95f-4292-a109-b8fb11ed37ec)

On smaller screen the top image is not fully displayed.

daoodaba975 replied:
Yes I noticed the image cuts off when you resize it.
Maybe it's the iframe that makes it like that?
But I think the most important thing is that we are just the form that appears.

NDOY3M4N replied:
Ok. On bigger screen this issue doesn't occur.

daoodaba975 replied:
Yes indeed

NDOY3M4N reviewed (commented):
Nice job!

NDOY3M4N committed:
🛠️ :: add yarn files to gitignore

NDOY3M4N committed:
💄 :: fix the layout of PartnerSection on mobile

There was some horizontal overflow on smaller devices

NDOY3M4N committed:
💄 :: move the container styles in the pages

The container styles were in the `Layout` before and some page (like the **about page** or the **hacktoberfest page** needed more vertical spacing for their content.
I also did some refactoring in the `EventGallery` component 😉

daoodaba975 reviewed (commented):
Well !
